finder-app/writer.c

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file	writer.c
+ * @author	Ruchit Naik
+ * @date	01-22-2022
+ * @brief	Application writes the specific string entered from the command line
+ * 		to the file along with the path entered from the terminal.
+ */ 
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <syslog.h>
+
+/*********** PRIVATE MACROS *************/
+#define REQ_ARG	3
+/****************************************/
+
+int main(int argc, char **argv){
+	//Setup syslog logging for the utility using LOG_USER
+	openlog(NULL, 0, LOG_USER);
+
+	//Check valid number of arguments passed
+	if(argc != REQ_ARG){
+		syslog(LOG_ERR, "ERROR: Insufficient arguments passed. Enter valid arguments <file path><string>");
+		closelog();					//Close log
+		return 1;
+	}
+
+	char *filep = argv[1];
+	char *string = argv[2];
```

Comment:
@ruchitnaik suggestion here: Use a const char* to the argument with descriptive name.